docs/PUBLISHING.md

```diff
@@ -4,6 +4,28 @@ Publishing Etcher
 This is a small guide to package and publish Etcher to all supported operating
 systems.
 
+Release Types
+-------------
+
+Etcher supports **production** and **snapshot** releases. Each is published to
+a different S3 bucket, and production releases are code signed, while snapshot
+releases aren't and include a short git commit-hash as a build number. For
+example, `1.0.0-beta.19` is a production release, while `1.0.0-beta.19+531ab82`
+is a snapshot release.
```

Comment:
Um? I can't see any stable / unstable notes here?  (unless they're so unstable that they actually evaporated :wink: )